common/src/main/java/com/github/khanshoaib3/minecraft_access/features/BiomeIndicator.java

```diff
@@ -6,45 +6,35 @@
 import net.minecraft.client.resource.language.I18n;
 import net.minecraft.registry.entry.RegistryEntry;
 import net.minecraft.world.biome.Biome;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * Narrates the name of the biome when entering a different biome.
  */
 @Slf4j
 public class BiomeIndicator {
-    private String previousBiome;
+    @Nullable
+    private String previousBiome = null;
 
     public BiomeIndicator() {
-        previousBiome = "";
     }
```

Comment:
This can be removed entirely. Java will create a public no args constructor by default if there is none.